test/2.0/csv.spec.ts

```diff
@@ -395,6 +395,240 @@ test("validateRow tall conditionals", (t) => {
+    )
+  )
+
+  const algorithmWithInfoRow = {
+    ...basicRow,
+    "standard_charge | negotiated_algorithm": "85",
+    estimated_amount: "60",
+  }
+  const algorithmWithInfoErrors = validateRow(
+    algorithmWithInfoRow,
+    9,
+    columns,
+    false
+  )
+  t.is(algorithmWithInfoErrors.length, 0)
+  const algorithmMissingInfoRow = {
+    ...basicRow,
+    "standard_charge | negotiated_algorithm": "standard method function",
+    estimated_amount: "60",
+    payer_name: "",
+    plan_name: "",
+    "standard_charge | methodology": "",
+  }
+  const algorithmMissingInfoErrors = validateRow(
+    algorithmMissingInfoRow,
+    10,
+    columns,
+    false
+  )
+  t.is(algorithmMissingInfoErrors.length, 3)
+  t.assert(
+    algorithmMissingInfoErrors[0].message.includes(
+      '"payer_name" is required when a payer specific negotiated charge is encoded as a dollar amount, percentage, or algorithm'
+    )
+  )
+  t.assert(
+    algorithmMissingInfoErrors[1].message.includes(
+      '"plan_name" is required when a payer specific negotiated charge is encoded as a dollar amount, percentage, or algorithm'
+    )
+  )
+  t.assert(
+    algorithmMissingInfoErrors[2].message.includes(
+      '"standard_charge | methodology" is required when a payer specific negotiated charge is encoded as a dollar amount, percentage, or algorithm'
+    )
+  )
+
+  // If the "standard charge methodology" encoded value is "other", there must be a corresponding explanation found
+  // in the "additional notes" for the associated payer-specific negotiated charge.
+  const otherWithNotesRow = {
+    ...basicRow,
+    "standard_charge | negotiated_percentage": "80",
+    estimated_amount: "150",
+    "standard_charge | methodology": "other",
+    additional_generic_notes: "explanation of methodology",
+  }
+  const otherWithNotesErrors = validateRow(
+    otherWithNotesRow,
+    11,
+    columns,
+    false
+  )
+  t.is(otherWithNotesErrors.length, 0)
+  const otherWithoutNotesRow = {
+    ...basicRow,
+    "standard_charge | negotiated_percentage": "80",
+    estimated_amount: "150",
+    "standard_charge | methodology": "other",
+  }
+  const otherWithoutNotesErrors = validateRow(
+    otherWithoutNotesRow,
+    12,
+    columns,
+    false
+  )
+  t.is(otherWithoutNotesErrors.length, 1)
+  t.is(
+    otherWithoutNotesErrors[0].message,
+    'If the "standard charge methodology" encoded value is "other", there must be a corresponding explanation found in the "additional notes" for the associated payer-specific negotiated charge.'
+  )
+
+  // If an item or service is encoded, a corresponding valid value must be encoded for at least one of the following:
+  // "Gross Charge", "Discounted Cash Price", "Payer-Specific Negotiated Charge: Dollar Amount", "Payer-Specific Negotiated Charge: Percentage",
+  // "Payer-Specific Negotiated Charge: Algorithm".
+  const itemNoChargeRow = {
```

Comment:
Just documenting for observed behavior:

If one of the standard charges is populated with an illegal value and the others left blank, an error is thrown about the illegal value and the `ITEM_REQUIRES_CHARGE` isn't thrown.

I think this is fine.